modules/installation-vsphere-machines.adoc

```diff
@@ -157,26 +165,18 @@ $ export IPCFG="ip=<ip>::<gateway>:<netmask>:<hostname>:<iface>:none nameserver=
 ----
 $ export IPCFG="ip=192.168.100.101::192.168.100.254:255.255.255.0:::none nameserver=8.8.8.8"
 ----
-
-... Set the `guestinfo.afterburn.initrd.network-kargs` property before booting a VM from an OVA in vSphere:
-+
-[source,terminal]
-----
-$ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${IPCFG}"
-----
 +
-*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*. Ensure that your VM's CPU and memory reservation have the following values:
-**** Memory reservation value must be equal to its configured memory size.
-**** CPU reservation value must be at least the number of low latency virtual CPUs multiplied by the measured physical CPU speed.
```

Comment:
**** CPU reservation value must be at least the number of virtual CPUs multiplied by the measured physical CPU speed.

Reply:
Sorry i dont understand why CPU reservation has been removed, yes you can remove the refrence to low latency Please add back the above...

Reply:
Hi @Timdawson264 . This PR needs to move forward, so I've removed the CPU reservation statement. Please file a separate docs bug Jira if you want the statement added back, where we can also work together on a warning statement for the customer to proceed with this configuration.